cpp/include/cudf/fixed_point/detail/floating_conversion.hpp

```diff
@@ -53,21 +54,19 @@ CUDF_HOST_DEVICE inline constexpr int count_significant_bits(T value)
     return 128 - (__clzll(high_bits) + static_cast<int>(high_bits == 0) * __clzll(low_bits));
   }
 #else
-  // Undefined behavior to call __builtin_clzll() with zero in gcc and clang
   if (value == 0) { return 0; }
 
-  // TODO: Use `std::countl_zero` instead of `__builtin_clzXX` once we migrate to C++20
   if constexpr (std::is_same_v<T, uint64_t>) {
-    return 64 - __builtin_clzll(value);
+    return 64 - cuda::std::countl_zero(value);
```

Comment:
Not related to parquet decoders but I modified `__builtin_clzxx()` to `cuda::std::countl_zero()` here too for consistency.